c2corg_ui/__init__.py

```diff
@@ -66,3 +71,18 @@ def _add_static_view(config, name, path):
     )
     config.add_cache_buster(path, version_cache_buster)
     CACHE_PATH.append(name)
+
+class NotFound():
```

Comment:
Please move this into `c2corg_ui/views/__init__.py`. All views defined in the `views` package.